newIDE/app/src/ObjectEditor/Editors/TextEditor.js

```diff
@@ -152,49 +159,167 @@ export default class TextEditor extends React.Component<EditorProps, void> {
             </ButtonGroup>
           </Line>
         </ResponsiveLineStackLayout>
-        <Line>
-          <Column expand noMargin>
-            <ResourceSelector
-              project={project}
-              resourceManagementProps={resourceManagementProps}
-              resourcesLoader={ResourcesLoader}
-              resourceKind="font"
+        <ResourceSelector
+          project={project}
+          resourceManagementProps={resourceManagementProps}
+          resourcesLoader={ResourcesLoader}
+          resourceKind="font"
+          fullWidth
+          canBeReset
+          initialResourceName={textObjectConfiguration.getFontName()}
+          onChange={resourceName => {
+            textObjectConfiguration.setFontName(resourceName);
+            this.forceUpdate();
+          }}
+          floatingLabelText={<Trans>Choose a font</Trans>}
+          hintText={<Trans>Choose a font</Trans>}
+        />
+        <Line noMargin>
+          <SemiControlledTextField
+            floatingLabelText={<Trans>Initial text to display</Trans>}
+            floatingLabelFixed
+            id="text-object-initial-text"
+            commitOnBlur
+            translatableHintText={t`Enter the text to be displayed by the object`}
+            fullWidth
+            multiline
+            rows={8}
+            rowsMax={8}
+            value={textObjectConfiguration.getText()}
+            onChange={value => {
+              textObjectConfiguration.setText(value);
+              this.forceUpdate();
+              this.props.onSizeUpdated();
+            }}
+          />
+        </Line>
+        <Text size="block-title" noMargin>
+          <Trans>Outline</Trans>
+        </Text>
+        <Checkbox
+          label={<Trans>Enabled</Trans>}
+          checked={textObjectConfiguration.isOutlineEnabled()}
+          onCheck={(e, checked) => {
+            textObjectConfiguration.setOutlineEnabled(checked);
+            this.forceUpdate();
+          }}
+        />
+        <ResponsiveLineStackLayout noMargin>
```

Comment:
You might want to use noColumnMargin here

Reply:
And maybe above as well, I'll let you decide what alignment you prefer

Reply:
I've added them but I didn't notice any difference. What does it do?

Reply:
When the component is wrapped (on small screens), it removes the side margins.